pymatgen/io/lobster/outputs.py

```diff
@@ -709,26 +733,11 @@ def get_structure_with_charges(self, structure_filename):
             Structure Object with Mulliken and Loewdin charges as site properties.
         """
         struct = Structure.from_file(structure_filename)
-        Mulliken = self.Mulliken
-        Loewdin = self.Loewdin
-        site_properties = {"Mulliken Charges": Mulliken, "Loewdin Charges": Loewdin}
+        mulliken = self.mulliken
```

Comment:
Can you please keep the old names for a while and deprecate them with a warning? see https://dev.to/hckjck/python-deprecation-2mof

Reply:
Done.